be/src/runtime/memory/chunk_allocator.h

```diff
@@ -74,10 +74,19 @@ class ChunkAllocator {
     // Free chunk allocated from this allocator
     void free(const Chunk& chunk, MemTracker* tracker = nullptr);
 
+    // Transfer the memory ownership to the chunk allocator.
+    // If the chunk allocator is full, then free to the system.
+    // Note: make sure that the length of 'data' is equal to size,
+    // otherwise the capacity of chunk allocator will be wrong.
+    void free_as_chunk(uint8_t* data, size_t size, MemTracker* tracker = nullptr);
```

Comment:
rename it to free(uint8_t* data, size_t size, MemTracker* tracker = nullptr)

Reply:
done